### ISSUE TITLE
Print ModSecurity version from nginx connector

### ISSUE THREAD

**Issue by theseion** [owasp-modsecurity/ModSecurity-nginx#316]
With the nginx connector (v3), the log output currently looks like this:
```
2024/04/04 16:11:33 [notice] 630#630: ModSecurity-nginx v1.0.3 (rules loaded inline/local/remote: 0/824/0)
2024/04/04 16:11:37 [notice] 1#1: ModSecurity-nginx v1.0.3 (rules loaded inline/local/remote: 0/824/0)
2024/04/04 16:11:37 [notice] 1#1: using the "epoll" event method
2024/04/04 16:11:37 [notice] 1#1: nginx/1.25.3
2024/04/04 16:11:37 [notice] 1#1: built by gcc 12.2.0 (Debian 12.2.0-14) 
2024/04/04 16:11:37 [notice] 1#1: OS: Linux 6.1.0-18-amd64
```

It would be helpful to also log the ModSecurity version, not only the connector version.

**Comment by airween** [comment]
Thanks, I think that's a really useful feature request.

What do you think, what would be the better solution: make a new, separated line, or would be fine to append to the first line (before the part `(rules loaded...`)?

**Comment by theseion** [comment]
I think it's fine to append it to the same line. The ModSec and connector versions belong together anyway.
Maybe prepend it though: `ModSecurity v3.0.12, ModSecurity-nginx v1.0.3...`.

**Comment by airween** [comment]
Please take a review: https://github.com/owasp-modsecurity/ModSecurity-nginx/pull/319.

**Comment by theseion** [comment]
Fixed by #319.
